src/utils/attribution-enhanced.ts

```
@@ -0,0 +1,435 @@
+    
+    switch (model) {
+      case AttributionModel.FIRST_CLICK:
+        credits[attribution.utm_source || 'direct'] = totalValue
+        break
+        
+      case AttributionModel.LAST_CLICK:
+        credits[attribution.utm_source || 'direct'] = totalValue
+        break
+        
+      case AttributionModel.LINEAR:
+        const touchpoints = this.getTouchpoints(sessionId)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)